volunteer.html

```diff
@@ -67,6 +67,44 @@ <h3 class="text-center">Development</h3>
     </div>
   </div>
 </div>
+<div class="panel">
+  <div class="panel-container">
+    <div class="fixed-width">
+      <h2 class="text-center">Testimonials</h2>
+      <div class="row">
+        <div class="column s-4-12">
+          <blockquote class="text-italic">
+            After 1 kickstart event, I was hooked, and haven’t looked back since! After finishing college, I just had to become a blueshirt, to help those like me realise their development potential too.
+          </blockquote>
+          <p class="text-heavy text-center">- Jake Howard</p>
+          <h5 class="text-center">
+            Team CLY SR2014-15<br>
+            Blue Shirt SR2016-21
+          </h5>
+        </div>
+        <div class="column s-4-12">
+          <blockquote class="text-italic">
+            SR was the main reason I signed up to do electronics at university, the experience gained by mentoring and coaching on tech days has been invaluable.
+          </blockquote>
+          <p class="text-heavy text-center">- Tom Wheal</p>
+          <h5 class="text-center">
+            Team PSC SR2014-15<br>
+            Blue Shirt SR2017-21
+          </h5>
+        </div>
+        <div class="column s-4-12">
+          <blockquote class="text-italic">
+            Volunteering with Student Robotics gave me the opportunity to work on projects with real deadlines. When applying for my first professional jobs, that experience let me stand out from other applicants.
+          </blockquote>
+          <p class="text-heavy text-center">- Peter Law</p>
+          <h5 class="text-center">
+            Blue Shirt SR2007-21
```

Comment:
Is "Blue Shirt" the term we want to use public-facing? It has a lot of context for us (and when you're actually at an event), but it's not the clearest term.
I realise we do use it elsewhere on the page (though as a single word, not two), though maybe we should reconsider that too?

Separately is using SR years for volunteer timeframes really useful? Again they don't feel entirely clear to the public and they don't really work well for people (such as myself) whose volunteering timeframes don't line up with the competition years. I suggest using calendar years instead.

Finally, including an end date on these kinda implies that the people mentioned have stopped being volunteers. Perhaps something like "since 2006" would be clearer?

Reply:
I've changed the captions underneath to be "volunteer" but have kept the term "blueshirt" in one testimonial and an instance above. I think it's a good term to keep if we can make it clear what's meant somewhere on the page

Reply:
https://github.com/srobo/website/pull/277/commits/0429890e1fca01e4dbe21f6bfb753e329abb2c71